### DISCUSSION TITLE
Details about Full System Restore after Setup Wizard

### DISCUSSION THREAD

**Discussion by tpreclik** [seedvault-app/seedvault#611] [Q&A]
I am currently stranded with a seedvault backup and a phone which is way past setup wizard (CalyxOS on Shift6mq). I would like to copy the seedvault backup to the phone and restore the apps. As I understand restoring all apps is the only option for now. I will also have to clear seedvault's app data since I have already setup a backup on the new phone (which is ok if it is deleted temporarily). But I am now hesitating for two reasons:

1. seedvault-extractor shows quite a few `com.android.*` apps (which I intentionally have not marked for backup when creating the backup on another phone) which could likely lead to unwanted side effects (for example contacts being removed) and
2. I wonder whether any apps will be removed when performing a full system restore (via vanity number `*#*#RESTORE#*#*`) or will the rest of the system (all apps not in backup) remain untouched?

Could you please clarify how the full system restore behaves?

**Comment by grote** [comment]
Apps should not get removed during restore, but app data gets overridden with what was backed up. When an app didn't have restore data in backup, it will not get touched.
